app/distributions/management/commands/stac_sync.py

```diff
@@ -228,6 +285,11 @@ def add_arguments(self, parser: CommandParser) -> None:
             default="",
             help="Add packages with missing dataset to this dataset"
         )
+        parser.add_argument(
+            "--create-default-dataset",
+            action="store_true",
+            help="Create the default dataset if needed"
+        )
```

Comment:
I imagine we always (or mostly) want to create these default datasets, no? We could omit the option (or invert to opt-out) and by default just create the default objects that are needed.